zetaclient/config/config.go

```diff
@@ -83,3 +93,12 @@ func GetPath(inputPath string) string {
 	}
 	return filepath.Join(path...)
 }
+
+func AnyBannedAddress(addrs ...string) bool {
```

Comment:
Can we add function comment?

Reply:
https://github.com/zeta-chain/node/pull/1790/commits/cf72c1bfc036951b792b8d3d7b327fbb7b53cc32